src/api/genres.js

```diff
@@ -98,3 +98,20 @@ export function destroyEmpty(auth) {
         : request.json().then((result) => Promise.reject(result));
     });
 }
+
+export function merge(auth, oldID, newID) {
```

Comment:
I think switching the order of these arguments makes sense. For people who know how the API is implemented, that order is the most logical order (or at least it is to me).